crates/core/src/rpc/admin.rs

```diff
@@ -798,7 +854,9 @@ impl AdminRpc for SurfpoolAdminRpc {
         let Ok(endpoint_url) = rx.recv_timeout(Duration::from_secs(10)) else {
             return Box::pin(async move { Err(jsonrpc_core::Error::internal_error()) });
         };
-        Box::pin(async move { Ok(endpoint_url) })
+        // Return a JSON string containing both UUID and endpoint URL
+        let response = format!(r#"{{"uuid": "{}", "endpoint": "{}"}}"#, uuid, endpoint_url);
```

Comment:
Previously, the return result would yield the TUI logging:
<img width="610" height="40" alt="image" src="https://github.com/user-attachments/assets/f6e6dedc-a163-4b0c-b593-b30774f88290" />

Now, the TUI logs this, which doesn't really make much sense:
<img width="978" height="42" alt="image" src="https://github.com/user-attachments/assets/b3e7d9f6-f263-496d-b22a-aeeb5a2c2a77" />

If we're going to log the UUID, it should be via `simnet_events_tx.try_send(SimnetEvent::info(...))`, and return the endpoint_url for this fn